src/components/Notification.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react'
+import { Alert, Snackbar } from '@mui/material'
+import Slide, { SlideProps } from '@mui/material/Slide'
+import { useAppDispatch, useAppSelector } from '../hooks/reduxHooks'
+import {
+    getNotificationMessage,
+    getNotificationSeverity,
+    getNotificationStatus,
+    updateNotificationStatus,
+} from '../redux/slices/app-config'
+
+type TransitionProps = Omit<SlideProps, 'direction'>
+
+function TransitionUp(props: TransitionProps) {
+    return <Slide {...props} direction="left" />
+}
+
+export default function Notifications() {
+    const notifStatus = useAppSelector(getNotificationStatus)
+    const notifMessage = useAppSelector(getNotificationMessage)
+    const notifSeverity = useAppSelector(getNotificationSeverity)
```

Comment:
I, personally would just use full wording `notificationStatus` and such, but it's just a suggestion, not a requirement

Reply:
@Chriserus  We usually use **get** or **set** to signify what's the purpose of the action, so I think this is a correct usage of the **get** notation.

Reply:
I wasn't clear, I don't like `notifSeverity`, I would prefer `notificationSeverity`, same for other consts @Ayagoumi

Reply:
@Chriserus this we agree upon, @aeddaqqa

Reply:
fixed